lib/hanami/router.rb

```diff
@@ -243,17 +243,17 @@ def self.define(&blk)
     #   [200, {}, ["{:name=>\"LG\",:id=>\"1\"}"]]
     #
     # rubocop:disable Metrics/MethodLength
-    def initialize(scheme: "http", host: "localhost", port: 80, prefix: "", namespace: nil, configuration: nil, finder: Routing::Endpoint, inflector: Dry::Inflector.new, not_found: NOT_FOUND, not_allowed: NOT_ALLOWED, &blk)
-      @routes        = []
-      @named         = {}
-      @namespace     = namespace
-      @configuration = configuration
-      @base          = Routing::Uri.build(scheme: scheme, host: host, port: port)
-      @prefix        = Utils::PathPrefix.new(prefix)
-      @finder        = finder
-      @inflector     = inflector
-      @not_found     = not_found
-      @not_allowed   = not_allowed
```

Comment:
Are the aligned assignments really worth the diff churn here? (I know they were aligned previously, so just one line being out of whack would be kind of strange, but I ask because I wonder if this is an opportunity to remove the alignment and reduce diff churn in the future). I realise this may be a core hanami "style guide" issue I'm touching on here, and if that's actually the case, your answer would help me ensure I follow suit in future :)

Reply:
I like this style. 😸